libx52/x52_control.c

```diff
@@ -22,7 +22,7 @@
 int libx52_vendor_command(libx52_device *x52, uint16_t index, uint16_t value)
 {
     int j;
-    int rc;
+    int rc = LIBUSB_ERROR_OTHER;
```

Comment:
I don't think this is really necessary, since `rc` is initialized at least once within the loop. I'd like to see the CFLAGS with which you compiled that threw this warning.